.github/workflows/prs-closed.yaml

```diff
@@ -18,6 +18,7 @@ jobs:
 
     steps:
       - name: Delete Website Bucket
-        uses: rishabhrao/s3-website-https-pr-action@v1
+        uses: brokenhandsio/s3-website-pr-action@v2
         with:
-          bucket-prefix: "vapor-blog-pull"
\ No newline at end of file
+          bucket-prefix: "vapor-blog-pull"
+          bucket-region: "us-east-1"
```

Comment:
Isn't this `eu-west-1`?

Reply:
The blog is actually in `eu-west-2` but it doesn't matter where the PRs go, as long as it's the same between the open and close workflows